test/Conversion/HandshakeToFIRRTL/test_buffer.mlir

```diff
@@ -139,3 +139,19 @@ handshake.func @test_buffer_init(%arg0: index, %arg1: none, ...) -> (index, none
   %0 = buffer [1] seq %arg0 {initValues=[42]} : index
   return %0, %arg1 : index, none
 }
+
+// -----
+
+// CHECK: %[[WIRE:.*]] = firrtl.wire  : !firrtl.bundle<field0: uint<32>, field1: uint<32>>
```

Comment:
The following checks are inside the buffer module, right? If so, please add a `CHECK-LABEL` on this just to make sure.